whipper/test/test_common_config.py

```diff
@@ -66,3 +66,28 @@ def testDefeatsCache(self):
         defeats = self._config.getDefeatsCache(
             'PLEXTOR ', 'DVDR   PX-L890SA', '1.05')
         self.assertEquals(defeats, True)
+
+    def test_get_musicbrainz_server(self):
+        # Test default return value
+        self.assertEquals(self._config.get_musicbrainz_server(),
+                          'musicbrainz.org')
```

Comment:
I think it would be worth adding a msg= param to assertEquals to say that this a test of the default value.